src/components/ResourceDetails.js

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from 'react';
+import { allCoursesList } from '../data/resourcesData';
+import { Link } from 'react-router-dom';
+
+const ResourceDetails = (props) => {
+	// const item ;
+	// const urlSearch = props.location.search;
+	// const params = new URLSearchParams(urlSearch);
+	// const
```

Comment:
please remove all comments that are not relevant to functionality and remove all console logs